### PR TITLE
Fix UEFI GDB and get/set variable.

### PR DESCRIPTION
**Pull request by liba2k**

### CODE REVIEW

File: qiling/os/uefi/uefi.py [hunk 1]
```diff
@@ -16,6 +16,7 @@ def __init__(self, ql):
         super(QlOsUefi, self).__init__(ql)
         self.ql = ql
         self.entry_point = 0
+        self.pid = 0
```
Comment (xwings) [inline qiling/os/uefi/uefi.py:19]: i think we got a OS wide pid. can u use that ?

Comment (xwings) [reply qiling/os/uefi/uefi.py:19]: my bad it was for posix. i will move it to QlOS so u can use it

Comment (xwings) [reply qiling/os/uefi/uefi.py:19]: Done moving, will be be able change it ?

Comment (liba2k) [reply qiling/os/uefi/uefi.py]: Done. I've added a small change in pe_uefi.

**Comment by xwings** [comment]
cool, thanks !

**Pull request merged**